hyp3_sdk/hyp3.py

```diff
@@ -120,71 +120,134 @@ def _refresh_batch(self, batch: Batch):
     def _refresh_job(self, job: Job):
         return self._get_job_by_id(job.job_id)
 
-    def submit_job_dict(self, job_dict: dict, name: Optional[str] = None, validate_only: bool = False) -> Job:
-        if name is not None:
-            if len(name) > 20:
-                raise ValidationError('Job name too long; must be less than 20 characters')
-            job_dict['name'] = name
+    def submit_prepared_jobs(self, prepared_jobs: Union[dict, List[dict]]) -> Batch:
+        """Submit a prepared job dictionary, or list of prepared job dictionaries
+
+        Args:
+            prepared_jobs: A prepared job dictionary, or list of prepared job dictionaries
+
+        Returns:
+            A Batch object containing the submitted job(s)
+        """
+        if isinstance(prepared_jobs, dict):
```

Comment:
could do 
```
if isinstance(propared_jobs, dict):
    prepared_jobs = [prepared_jobs]
payload = {'jobs':prepared_jobs}
```
Not conviced it is actually better though

Reply:
Think I'm going to leave it... no strong argument here either way